src/sagemaker/estimator.py

```diff
@@ -50,7 +50,8 @@ class EstimatorBase(with_metaclass(ABCMeta, object)):
     def __init__(self, role, train_instance_count, train_instance_type,
                  train_volume_size=30, train_volume_kms_key=None, train_max_run=24 * 60 * 60, input_mode='File',
                  output_path=None, output_kms_key=None, base_job_name=None, sagemaker_session=None, tags=None,
-                 subnets=None, security_group_ids=None, model_uri=None, model_channel_name='model'):
+                 subnets=None, security_group_ids=None, metric_definitions=None, model_uri=None,
+                 model_channel_name='model'):
```

Comment:
let's add `metric_definitions` to the end of the list (to prevent a breaking change)

Reply:
Fixed